raiden/transfer/channel.py

```diff
@@ -978,6 +978,15 @@ def get_batch_unlock_gain(channel_state: NettingChannelState,) -> UnlockGain:
     )
 
 
+def get_capacity(channel_state: NettingChannelState) -> TokenAmount:
+    return TokenAmount(
```

Comment:
perhaps add a docstring with a link to the capacity definition here?

https://raiden-network.readthedocs.io/en/stable/glossary.html#term-channel-capacity